train.py

```diff
@@ -1191,10 +1222,9 @@ def _validate(data_loader, model, criterion, loggers, args, epoch=-1, tflogger=N
                 # correct output for accurate loss calculation
                 if args.act_mode_8bit:
                     output /= 128.
-                    for key in model.__dict__['_modules'].keys():
-                        if (hasattr(model.__dict__['_modules'][key], 'wide')
-                                and model.__dict__['_modules'][key].wide):
-                            output /= 256.
+                    for _, module in model.named_modules():
+                        if hasattr(module, 'wide') and module.wide:
+                            output /= 128.
```

Comment:
Q: Was the previous version (division by 256) wrong?

Reply:
Yes, it was. The resolution for wide outputs is 14 bits, which requires division by 128 here instead of 256.